### ISSUE TITLE
Conversations not being marked as read

### ISSUE THREAD

**Issue by BWMerlin** [dm0-/window_merge#6]
With message notification (default pidgin plugin) enabled as well as window_merg when a conversation has focus it is not automatically marked as being read (the highlighted conversation tab doesn't lose is bold and colour change).

For example, I have pidgin running as a none focused window while I type this ticket out, if I go back to pidgin if any new text was typed into a conversation window and that tab is the currently active window it will get marked as being read straight away.  But if I enable window_merge this does not happen, I have activate a different conversation tab and then go back to the original tab for that conversation to be marked as read.

Basically when new text is typed into a conversation tab the big bold red text on the conversation tabs name isn't being cleared as it would if window_merge wasn't being used.

**Comment by dm0-** [comment]
Thanks for reporting.

After looking into this at first, it didn't seem there was a clean way to fix it since this is another case of the relevant functions being static in Pidgin and unusable to plugins.  However, I saw a fix was possible in my experimental tiling-compatibility branch.

I spent a little while cherry-picking the necessary changes (and some others that didn't seem too volatile) and pushed all that to GitHub.  There are a handful of other minor bug fixes in there as well.  In addition, I rebuilt the Windows plugin on the download page again with these changes included.

I will close this ticket for now, as the issue appears to be corrected for me.  Please give it a testing and let me know if there are still problems with it.
